src/App/App.tsx

```diff
@@ -65,197 +65,195 @@ class App extends Component {
       <Container fluid={true}>
         <Helmet titleTemplate={`%s | ` + WEBSITE_NAME} defaultTitle="" />
         <ConnectedHeader />
-        <Container>
```

Comment:
This line was removed

Reply:
Instead of removing it, would it be possible to define a width for it other than the default (I think it might be 1140px or something like that)?

Reply:
@moshthepitt what is the value of keeping it and re-defining it as fluid?

Reply:
Good catch @moshthepitt thanks